### ISSUE TITLE
Not works with my htaccess

### ISSUE THREAD

**Issue by solutac** [MattWilcox/Adaptive-Images#192]
I have to use htaccess to redirect requests to application root, which is in **/public** from webserver root. There is bootstrap of application which proccess all requests. So I use this htaccess:

```
RewriteEngine on
RewriteRule ^\.htaccess$ - [F]
RewriteCond %{REQUEST_URI} =""
RewriteRule ^.*$ /public/index.php [NC,L]
RewriteCond %{REQUEST_URI} !^/public/.*$
RewriteRule ^(.*)$ /public/$1
RewriteCond %{REQUEST_FILENAME} -f
RewriteRule ^.*$ - [NC,L]
RewriteRule ^public/.*$ /public/index.php [NC,L]
```

I tried add these lines to htaccess:
```
RewriteCond %{REQUEST_URI} !/public/ai-cache
RewriteCond %{REQUEST_URI} /public/images
#RewriteRule \.(?:jpe?g)$ /public/adaptive-images.php
```

or create new htaccess saved in **/public** folder:
```
RewriteEngine on
RewriteCond %{REQUEST_URI} !ai-cache
RewriteCond %{REQUEST_URI} images
RewriteRule \.(?:jpe?g)$ adaptive-images.php
```

but nothing works. Cache folder has enough permissions, but no images are saved there and firebug console alert, that images on the page does not exist.

It is possible to debug in some way? Or can I merge my htaccess and Adaptive images one together?

**Comment by staabm** [comment]
Does the your apache in generall accept htaccess settings in this directory?

**Comment by solutac** [comment]
Yes it accepts. But I would like to merge it rather. I do not know if images are redirect to adaptive-images.php. How to find out?

**Comment by staabm** [comment]
Activate apache access-log and/or rewrite log

**Comment by solutac** [comment]
I have activated access log (rewrite log is not possible to manage in customer centre of my webhosting), but there is nothing. Adaptive images is amazing tool, I tested on localhost and it works like charm. But in this case (redirecting of root), which is quite useall, not works. There have to be some way how to merge those two htaccesses.
